_locales/th/messages.json

```diff
@@ -0,0 +1,18 @@
+{
+    "appName": {
+        "message": "ตัวช่วยในการเช็ค Pull Request",
```

Comment:
Thanks for the PR. This should reference BitBucket since this extension only works on that site.

Reply:
I'm just gonna go with it. Thanks again!